torch/csrc/autograd/python_engine.cpp

```diff
@@ -27,7 +27,6 @@ static torch::autograd::python::PythonEngine engine;
 // throughout the code in both NO_PYTHON builds and regular builds
 Engine& torch::autograd::Engine::get_default_engine() {
   return engine;
-}
```

Comment:
Missing bracket is causing all the tests to fail.

Reply:
yes, I mistakely deleted it when resolving conflict